Headers/phot_container_mod.F90

```diff
@@ -598,7 +602,7 @@ SUBROUTINE Init_Phot_Container( Input_Opt, State_Grid, Phot, RC )
        Phot%ASYMAA = 0d0
     
        ! Phot%PHAA      (:,:,:,:)
-       ALLOCATE( Phot%PHAA( Phot%NWVAA, Phot%NRAA, Phot%NSPAA, 8 ), &
+       ALLOCATE( Phot%PHAA( Phot%NWVAA, Phot%NRAA, Phot%NSPAA, 8, Phot%NDRg ), &
```

Comment:
Are these arrays also deallocated in this module?

Reply:
They are.

Reply:
@Haihui-Zhu Thanks!